### ISSUE TITLE
File upload broken by huge images

### ISSUE THREAD

**Issue by bergwerf** [marcantondahmen/automad#85]
I didn't investigate this in detail, but I uploaded a 12000x12000 pixel image by accident which broke the file uploading interface (`General Data and Files`). The `FileCollection::edit` and `Image::select` returned a 503, and the uploaded files were no longer displayed. I suspect the script crashes when trying to determine the dimensions of that one huge image. Perhaps worth investigating. The issue resolved itself after I manually deleted that huge image from the `/shared` directory.

**Comment by marcantondahmen** [comment]
Hi, thanks for reporting this. As far as I understand this could have two causes. It is either the PHP memory limit or the PHP maximum file and post size. 

In both cases it can be resolved by configuring PHP correctly, allowing larger files to be uploaded and processed.

The default `.htaccess` includes an example configuration for large files at the top that can be uncommented. This might be already enough.

**Comment by marcantondahmen** [comment]
Yeah, I agree. A proper error message would make totally sense! Let me know if increasing the PHP settings helps.
